src/components/generic/movies/movies.tsx

```diff
@@ -21,7 +20,7 @@ const placeholderStyle = {
   borderRadius: '5px',
 }
 
-const MoviesPanel: FC<Props> = ({movies, loader, loading}) => {
+export const MoviesPanel: FC<Props> = ({movies, loader, loading}) => {
```

Comment:
It would be best to rename this file to `MoviesPanel.tsx`. It's a good practice to have the same file names as component names.

Reply:
I changed it. You're right.